browser/src/map/Clipboard.js

```diff
@@ -581,6 +581,9 @@ L.Clipboard = L.Class.extend({
 		if ($('.w2ui-input').is(':focus'))
 			return true;
 
+		if ($('input.ui-combobox-content').is(':focus'))
+			return true;
```

Comment:
good enough for now, we need to look for our helper to detect inputs and unify in all places later